File: app/src/main/java/eu/toma/dev/playground/warcraft/model/Spec.java
package eu.toma.dev.playground.warcraft.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Spec
{
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("role")
    @Expose
    public String role;
    @SerializedName("backgroundImage")
    @Expose
    public String backgroundImage;
    @SerializedName("icon")
    @Expose
    public String icon;
    @SerializedName("description")
    @Expose
    public String description;
    @SerializedName("order")
    @Expose
    public Integer order;
}

File: app/src/main/kotlin/warcraft/model/Spec.kt
package warcraft.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Spec {
    @SerializedName("name")
    @Expose
    lateinit var name: String

    @SerializedName("role")
    @Expose
    lateinit var role: String

    @SerializedName("backgroundImage")
    @Expose
    lateinit var backgroundImage: String

    @SerializedName("icon")
    @Expose
    lateinit var icon: String

    @SerializedName("description")
    @Expose
    lateinit var description: String

    @SerializedName("order")
    @Expose
    var order: Int? = null
}
